src/rdkafka_fetcher.c

```diff
@@ -387,7 +387,7 @@ rd_kafka_fetch_reply_handle_partition(rd_kafka_broker_t *rkb,
                 if (rd_kafka_buf_ApiVersion(request) >= 5)
                         rd_kafka_buf_read_i64(rkbuf, &hdr.LogStartOffset);
 
-                rd_kafka_buf_read_i32(rkbuf, &AbortedTxnCnt);
+                rd_kafka_buf_read_arraycnt(rkbuf, &AbortedTxnCnt, -1);
```

Comment:
```suggestion
                rd_kafka_buf_read_arraycnt(rkbuf, &AbortedTxnCnt, RD_KAFKAP_ABORTED_TRANSACTIONS_MAX);
```
Add a new define RD_KAFKAP_ABORTED_TRANSACTIONS_MAX after RD_KAFKAP_CONFIGS_MAX with a value of 1000000.
Also you can remove this condition later:
```
if (unlikely(AbortedTxnCnt > 1000000))
```